samples/oneapi/dpc/ccl/makefile_lnx

```diff
@@ -28,6 +28,8 @@ help:
 	@echo "                    for Intel(R) oneAPI DPC++/C++ Compiler."
 	@echo
 	@echo "mode_name         - can be build or run. Default is run"
+	@echo
+	@echo "proc              - number of processes (GPUs). Default is 1"
```

Comment:
Default is 1? Later it's assigned to 4

Reply:
Corrected the help message. The parameter itself is set to 4, as it was before.